data/json/items/generic.json

```diff
@@ -2496,5 +2496,82 @@
     "symbol": ".",
     "color": "white",
     "ammo_type": "components"
+  },
+  {
+    "id": "hp_compressor",
+    "copy-from": "pump_complex",
```

Comment:
Why do we need another high pressure pump with slightly different characteristics? To be able to craft it, as existing high pressure pump isn't craftable on purpose?

Reply:
Definitely no. This one is electric compressor, that uses existing (mechanic) one, which is not craftable as you notice, in it's own recipe, if you look closely.